src/main/scala/beam/router/r5/R5Wrapper.scala

```diff
@@ -305,18 +307,20 @@ class R5Wrapper(workerParams: R5Parameters, travelTime: TravelTime, travelTimeNo
         geo.utm2Wgs(request.destinationUTM),
         10E3
       )
-      val vehicleLegMode = vehicle.mode.r5Mode.flatMap(_.left.toOption).getOrElse(LegMode.valueOf(""))
+      val directMode = vehicle.mode.r5Mode.get.left.get
```

Comment:
This change was done in last PR. We have to change becuase of TryGet error so now if we revert this again as original then we will encountered again TryGet error

Reply:
reverted